extra/callback_server.py

```diff
@@ -0,0 +1,30 @@
+from http.server import HTTPServer, BaseHTTPRequestHandler
+import sys
+
+
+class RequestHandler(BaseHTTPRequestHandler):
+    def do_POST(self):
```

Comment:
rename the function to something a bit clearer

Reply:
I'm afraid that's not possible, this is the function we override to handle post requests, as defined in native python docs for this library. more info here: https://docs.python.org/3/library/http.server.html

Reply:
Thanks I hate it lol